utilities/src/biginteger/bigint_256.rs

```diff
@@ -265,7 +265,11 @@ impl FromBits for BigInteger256 {
 impl ToBytes for BigInteger256 {
     #[inline]
     fn write_le<W: Write>(&self, writer: W) -> IoResult<()> {
-        self.0.write_le(writer)
+        let mut arr = [0u8; 8 * 4];
+        for (i, num) in self.0.iter().enumerate() {
+            arr[i * 8..][..8].copy_from_slice(&num.to_le_bytes());
```

Comment:
Instead of `arr[i * 8..][..8]`, can you write `arr[i * 8..(i + 1) * 8]`?

Reply:
sure, done in https://github.com/AleoHQ/snarkVM/pull/1884/commits/b4f9aa2c3887f5b7b3e6ee068e3dea3eccbd98b1